pkgs/development/interpreters/ruby/default.nix

```diff
@@ -268,9 +268,9 @@ in {
   };
 
   ruby_3_1 = generic {
-    version = rubyVersion "3" "1" "0" "";
+    version = rubyVersion "3" "1" "1" "";
     sha256 = {
-      src = "sha256-UKBQTG7ctNYc5rjP292qlXBxlfqw7Ne16SZUsqlBKFQ=";
+      src = "sha256-/m5Hgt6XRDl43bqLpL440iKqJNw+PwKmqOdwHA7rYZ0=";
       git = "sha256-TcsoWY+zVZeue1/ypV1L0WERp1UVK35WtVtYPYiJh4c=";
```

Comment:
What is this, the checksum of the git revision of 3.1.1?

Reply:
This is outstanding to fix as I cannot figure this out, and it probably is the cause why the ruby version is not updated.

```
% nix-build "$(pwd)" -A ruby_3_1 && ./result/bin/ruby -v
/nix/store/az6fr4svzibnn3sa4x4nhidrdwy3vy9k-ruby-3.1.1
ruby 3.1.0p0 (2021-12-25) [x86_64-darwin21]
```

Or - I'm not rebuilding the correct version of the package. The command above is run inside this branch.

Reply:
That was it

```
% ./result/bin/ruby --version
ruby 3.1.1p18 (2022-02-18) [x86_64-darwin21]
```